lib/pf/StatsD.pm

```diff
@@ -148,12 +147,6 @@ sub update {
     my ( $self, $stats, $delta, $sample_rate ) = @_;
     $delta = 1 unless defined $delta;
     my %data;
```

Comment:
The data hash is empty!

Reply:
That's what makes the graphs so beautiful now ;)